src/openapi/peridio-admin-openapi.yaml

```
@@ -4550,6 +4550,14 @@ components:
                   $ref: "#/components/schemas/device-event-certificate"
                 ca_certificate:
                   $ref: "#/components/schemas/device-event-ca-certificate"
+                route:
+                  type: string
+                  description: |
+                    The URL route where the device connected
+
+                    This could be either `/socket/websocket` for websocket connections
+                    or an URL path of the Device API
```

```suggestion
```suggestion
                    The route, i.e., the URL path to which the device connected.
```

Pedantic formalization of the first sentence, plus dropping the second part altogether because I don't want to have to maintain an enumerated list of literal content values as they change over time.
```